docker_push_latest_if_changed.py

```diff
@@ -1,5 +1,182 @@
 #!/usr/bin/env python3
 import argparse
+import hashlib
+import logging
+from collections import namedtuple
+
+import docker
+
+
+ImageParityFields = namedtuple(
+    'ImageParityFields',
+    ['image_cmds_hash', 'packages_hash'],
```

Comment:
when equivalent, use tuples: https://github.com/asottile/scratch/wiki/Prefer-tuples-to-lists

Reply:
Sounds good to me.
Switched to tuples.